packages/botonic-nlp/src/parser/parser.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { lstatSync, readdirSync, readFileSync } from 'fs'
+import * as yaml from 'js-yaml'
+import { join } from 'path'
+
+import { DataAugmenter } from './data-augmenter'
+import { EntitiesParser } from './entities-parser'
+import { Data, Sample } from './types'
+
+export class Parser {
+  private static CLASS_FIELD = 'class'
+  private static ENTITIES_FIELD = 'entities'
+  private static DATA_AUGMENTATION_FIELD = 'data-augmentation'
+  private static SAMPLES_FIELD = 'samples'
+
+  static parse(path: string): Data {
+    const stat = lstatSync(path)
+    if (stat.isFile()) {
+      return Parser.parseFile(path)
+    } else if (stat.isDirectory()) {
+      return Parser.parseDirectory(path)
+    } else {
+      throw new Error(`path "${path}" must be a directory or a file.`)
```

Comment:
not needed else
if(stat.isFile()) return
if(stat.isDirectory()) return
throw New Error